scout/parse/hpo_terms.py

```diff
@@ -48,6 +48,10 @@ def parse_hpo_obo(hpo_lines):
         if len(line) == 0:
             continue
         line = line.rstrip()
+
+        if line == "[Typedef]":
+            break
```

Comment:
Isn't this setting up for problems if the sort order were to change? How about skipping a while until a new term is found?